Composer/packages/server/__tests__/models/bot/botProject.test.ts

```diff
@@ -228,11 +228,14 @@ describe('lu operation', () => {
     }
   });
 
-  it('should throw error when lu content is invalid', async () => {
+  it('should update diagnostics when lu content is invalid', async () => {
     const id = 'root';
     const content = 'hello \n hello3';
-
-    await expect(proj.updateLuFile(id, content)).rejects.toThrow();
+    const luFiles = await proj.updateLuFile(id, content);
+    const result = luFiles.find(f => f.id === id);
+    if (result !== undefined) {
+      await expect(result.diagnostics.length).toBeGreaterThan(0);
```

Comment:
This does not need to be `await`ed.

Reply:
You can also drop the `result !== undefined` and do `expect(result?.diagnostics?.length).toBeGreaterThan(0)` (typescript 3.7)

Reply:
I will upgrade typescript' version to 3.7.2 in package.json

Reply:
I found that you have already upgraded the version